src/frontend/src/eth/components/receive/EthReceive.svelte

```diff
@@ -23,5 +25,11 @@
 </script>
 
 <ReceiveButtonWithModal open={openReceive} isOpen={$modalEthReceive}>
-	<EthReceiveModal slot="modal" />
+	<ReceiveModal slot="modal" address={$networkAddress ?? undefined}>
```

Comment:
the `??` check is not needed here right?

Reply:
I adjusted the type of address prop so `??` is now gone.